plutus-core/plutus-core/src/PlutusCore/Parser/Builtin.hs

```diff
@@ -65,7 +65,7 @@ conText = do
 
 -- | Parser for unit.
 conUnit :: Parser (Some (ValueOf DefaultUni))
-conUnit = someValue () <$ symbol "()"
+conUnit = someValue () <$ (char '(' *> whitespace *> char ')')
```

Comment:
I think the intended way of doing this kind of thing would be `symbol "(" *> symbol ")"`, so that trailing whitespace is consumed too, but we don't maintain the invariant that parsers consume trailing whitespace, so your version makes more sense.

Reply:
Oh, I didn't notice that `symbol` preserves trailing whitespace. Then I think it should use `symbol` since it did before.